frontend/src/metabase/ui/components/inputs/SpecialTagsInput/SpecialTagsInput.tsx

```diff
@@ -132,6 +141,18 @@ export const SpecialTagsInput = forwardRef(function _SpecialTagsInput(
     onChange: onSearchChange,
   });
 
+  const getStyles = useStyles<TagsInputFactory>({
+    name: "TagsInput",
+    classes: {} as any,
```

Comment:
I know you copied this from mantine, but the type cast is not needed. Hoping to add `noExplicitAny` to our tsconfig some day, so please remove 🙏